### PR TITLE
changelog: Update changelog for 1.13.4

### PR DESCRIPTION
**Pull request by zmrow**
*Draft until the PRs in this changelog are merged*
**Issue number:**
N/A

**Description of changes:**
Update CHANGELOG for 1.13.4

**Testing done:**
N/A

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+
+* Generate and pass `--hostname-override` flag to kubelet in `aws-k8s-1.26` variants ([#3033])
+
+[#3032]: https://github.com/bottlerocket-os/bottlerocket/pull/3032
+[#3033]: https://github.com/bottlerocket-os/bottlerocket/pull/3033
+[#3037]: https://github.com/bottlerocket-os/bottlerocket/pull/3037
+
 # v1.13.3 (2023-04-17)
 
 ## OS Changes
```
Comment (etungsten) [inline CHANGELOG.md:20]: Like Ben mentioned, I think we should "fix" the 1.13.3 changelog real quick by moving

> * Generate 'provider-id' setting for aws-k8s variants ([#3026])

To the "Orchestrator Changes -> Kubernetes" section.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
+# v1.13.4 (2023-04-24)
+
+## OS Changes
+
+* Ensure the correct hostname is used when a VPC DHCP option set has multiple domains ([#3032])
```
Comment (bcressey) [inline CHANGELOG.md]: It's more that it's the first hostname, right? There's not really a "correct" one.

Comment (zmrow) [reply CHANGELOG.md]: Right - will fix

**Comment by zmrow** [comment]
^ Addresses @etungsten and @bcressey 's comments

**Review by etungsten — APPROVED** [review]

**Review by yeazelm — APPROVED** [review]

**Review by cbgbt — APPROVED** [review]
🚀 

The dependent PRs for this have been merged. Gonna pull this out of draft and merge it.

**Pull request merged**